src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala

```diff
@@ -72,9 +73,22 @@ object config {
         bad,
         args.optional("pii"),
         resolver,
-        args.optional("enrichments")
+        args.optional("enrichments"),
+        args.optional("labels").map(parseLabels)
       )
 
+    private[this] def parseLabels(input: String) = {
+      val labelDelimiter = ";"
+      val valueDelimiter = "="
+
+      input
+        .split(labelDelimiter)
+        .map(_.split(valueDelimiter))
+        .filter(_.length == 2)
```

Comment:
It would be great to not silently discard an error in the configuration.

Reply:
Agreed, but shouldn't this happen in `Enrich.main()` where we parse the config?